projects/graduated/riscv-dv.md

```diff
@@ -0,0 +1,40 @@
+# Project application: `riscv-dv`
+
+## Application
+
+### Sandbox application
+
+* Project name: `riscv-dv`
+* Project repo(s): https://github.com/google/riscv-dv
+* Brief summary of the project: `riscv-dv` is an open source verification tool for RISC-V processors. `riscv-dv` is a SystemVerilog based random RISC-V instruction generator that checks the execution against an industry standard ISS for correction and compliance.
+* Project's open source license: Apache 2.0
+* Link to issue tracker: https://github.com/google/riscv-dv/issues
+* Link to website: None.
+* Links to social media accounts: None.
+* Who uses this project, and at what scale: Google, OpenTitan, Western Digital, Andes, and others.
+* Why does the project want to join CHIPS Alliance: To enable the CHIPS Alliance community to design and validate RISC-V CPU Cores.
+* Primary contact from the project during the Sandbox application process:
+ * Name: Matt Cockrell
```

Comment:
Minor nit, the indentation is not proper markdown, these bullets will not be indented correctly (it's also used in some projects already in the repo so that was probably copied).